lib/common/strings.c

```diff
@@ -457,7 +457,8 @@ pcmk_parse_interval_spec(const char *input, guint *result_ms)
         crm_time_t *period_s = crm_time_parse_duration(input);
 
         if (period_s != NULL) {
-            msec = 1000 * crm_time_get_seconds(period_s);
+            msec = crm_time_get_seconds(period_s);
+            msec = QB_MIN(msec, G_MAXUINT / 1000) * 1000;
```

Comment:
would we exceed 80 chars when writing crm_time_get_seconds(period_s) directly in QB_MIN?
at least we would never write seconds in a variable that is called msec ;-)
and the casting is not the reason as it is already long long.
picky - was just thinking why you did it the way you did ...

Reply:
I'd assume the compiler optimizes that away - but yeah ...